src/components/legend/draw.js

```diff
@@ -426,7 +448,7 @@ function setupTraceToggle(g, gd) {
         }
     });
     traceToggle.on('mouseup', function() {
-        if(gd._dragged) return;
+        if(gd._dragged || gd._editing) return;
```

Comment:
I'll admit that blocking other interactions while the edit to go on was more of a secondary benefit to introducing this - the primary motivation was that it served as a way to prevent the `mouseup` event from having any real effect when the `click` fired from the `svgTextUtils.makeEditable` otherwise when you clicked on the text, it would both make the field editable and toggle the series.

When the traceToggle use to use `click` before my change, I'm not sure what was preventing both click handlers from firing when the label was clicked.

If there's a suggestion on how to prevent both actions from firing, I can remove this flag and leave the question on how interactions should be dealt with generally to another ticket.